src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailai-image-id']!;
+  const csrfToken = (
+    input.form!.querySelector(
+      'input[name=csrfmiddlewaretoken]',
+    ) as HTMLInputElement
```

Comment:
**Issue:** Could avoid casting by passing a generic type to `querySelector`, e.g. `querySelector<HTMLInputElement>("input[name=blabla]")`